index.html

```diff
@@ -14,14 +14,35 @@
       rel="stylesheet"
     />
 
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Community of developers who collaborate to build Real software."
+    />
+
+    <!-- Open Graph / Facebook Meta Tags -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.realdevsquad.com/index.html" />
     <meta property="og:title" content="Real Dev Squad" />
     <meta
       property="og:description"
       content="Community of developers who collaborate to build Real software."
     />
     <meta property="og:image" content="img/Real-Dev-Squad@2x.png" />
```

Comment:
Please use the image which I have provided in the description as it has white background which is needed for better preview.